MAINTAINERS

```diff
@@ -89,7 +89,7 @@
 			"albers",
 
 			# Andrea Luzzardi started contributing to the Docker codebase in the "dotCloud"
-			# era, even before it was called "Docker". He is one of the architects of both
+			# era, even before it was called "Docker". They is one of the architects of both
```

Comment:
For _general_ docs, this would be good, but for this specific file, the contributor/maintainer's preferred pronouns (at the time) were used.